1_fetch/src/get_site_info_data.R

```diff
@@ -1,17 +1,11 @@
-get_site_info_data <- function(eval_data, file_outpath){ 
+get_site_info_data <- function(eval_data){ 
   
-  # function to access and download site information based on unique site numbers from 2_process/out/all_nwisdata.csv
-  
-    # get site list
-      nwisdata <- readr::read_csv(eval_data)
-      sites <- unique(nwisdata$site_no) 
+  # function to access and download site information based on unique site numbers
   
     # download site info - loop through sites and append to a single dataframe
-      data_out <- retry(download_nwis_site_info(sites), maxErrors = 10, sleep = 2)
-    
-      readr::write_csv(data_out, file_outpath)
+      data_out <- retry(download_nwis_site_info(site_list), maxErrors = 10, sleep = 2)
```

Comment:
I don't see anything in this function called `site_list`. I think you still want to keep line 7 so that you can get a vector of `sites` from the data to use here. I think you could just update to `sites <- unique(eval_data$site_no)` since you no longer need to read in a CSV file.

Reply:
I created the `site_list` variable in `_targets.R`, but I updated this function to look at the unique site numbers in the combined file instead, as you suggested.

Reply:
Right, if `site_list` is in `_targets.R`, you should pass it _into_ this function as an argument. So, I would suggest declaring this function as `get_site_info_data <- function(eval_data, site_list)` and then changing your targets code that calls the function to pass in this second argument.

Reply:
Oh! Ok, this makes sense. Went ahead and did this and pushed those updates.